src/library/methods/R/RClassUtils.R

```diff
@@ -2412,7 +2412,7 @@ classesToAM <- function(classes, includeSubclasses = FALSE,
       maybe <- if(!is.null(f <- get0(x, where))) is.function(f) else FALSE
       if(maybe)
         maybe <- is(f, "genericFunction") ||
-              (length(grep("UseMethod", deparse(f))) > 0) ||
+              any(grepl("UseMethod", deparse(f))) ||
```

Comment:
```suggestion
              any(grepl("UseMethod", deparse(f), fixed = TRUE)) ||
```

Reply:
good catch! Agree it's an improvement. I don't have an easy workflow for updating patches, so I'll just make a note of it in the issue tracker.